Swabbr.Core/Storage/StorageObject.cs

```diff
@@ -0,0 +1,35 @@
+﻿namespace Swabbr.Core.Storage
+{
+    /// <summary>
+    ///     Storage object settings.
+    /// </summary>
+    public class StorageObject
```

Comment:
This was never finished in FM, most likely will change

Reply:
Noted, made issue #197